### DISCUSSION TITLE
Can Mergify allow to merge PRs that update .mergify.yml if a previous flag was added ?

### DISCUSSION THREAD

**Discussion by v1v** [Mergifyio/mergify#2821] [Q&A]
I've seen this https://github.com/Mergifyio/mergify-engine/discussions/2546#discussioncomment-702669 regarding `auto-merge` PRs that are changing `.mergify.yml`.

I understand the reason, though, is there any way to relax this particular restriction if the `.mergify.yml` already contains a flag that workaround this?

In other words, if `mergify.yml` was configured with a less relax merge policy in advance this could help with some automations.

Something like the below snippet could help to demonstrate my proposal:

```
## this will be false by default unless it was explicitly configured
allow_automerge: true
pull_request_rules:
```

What am I trying to solve?

- Branch strategy for one of the projects I work with is:
  - `master`
  - `<major>.x`  
  - `<major>.<minor>`

When a new minor branch is created then the backport rules in `.mergify.yml` are required to be changed accordingly.

**Comment by jd** [comment]
That sounds doable. I agree that it should be optional and disabled by default, but if that's what one wants, why not.
